probe/main.go

```diff
@@ -83,24 +83,22 @@ func main() {
 	)
 
 	// TODO provide an alternate implementation for Darwin.
-	if runtime.GOOS == linux {
-		processCache = process.NewCachingWalker(process.NewWalker(*procRoot))
-		reporters = append(reporters, process.NewReporter(processCache, hostID))
+	processCache = process.NewCachingWalker(process.NewWalker(*procRoot))
+	reporters = append(reporters, process.NewReporter(processCache, hostID))
 
-		if *dockerEnabled {
-			if err = report.AddLocalBridge(*dockerBridge); err != nil {
-				log.Fatalf("failed to get docker bridge address: %v", err)
-			}
-
-			dockerRegistry, err := docker.NewRegistry(*dockerInterval)
-			if err != nil {
-				log.Fatalf("failed to start docker registry: %v", err)
-			}
-			defer dockerRegistry.Stop()
+	if runtime.GOOS == linux && *dockerEnabled {
```

Comment:
Sorry wasn't clear.  Things like entrypoint.sh or even our scope script.  I'm assuming you don't use them when launching on Darwin.